README.md

```diff
@@ -142,6 +143,75 @@ The createTokenSynchronous method allows you to handle threading on your own, us
                     }
                 });
 
+#### IntentService Example
+
+You can invoke the following from your code (where `cardToSave` is some Card object that you have created.)
+
+        Intent tokenServiceIntent = TokenIntentService.createTokenIntent(
```

Comment:
Minor nit, but can you use the triple-` syntax instead of indentation? It gives some minor syntax highlighting if you specify the language.

Feel free to view/source on this.
```java
Intent tokenServiceIntent = TokenIntentService.createTokenIntent(
  mActivity,
  cardToSave.getNumber(),
  cardToSave.getExpMonth(),
  cardToSave.getExpYear(),
  cardToSave.getCVC(),
  mPublishableKey);
```